### ISSUE TITLE
Documentation fix of "Text Classification with Movie Reviews"

### ISSUE THREAD

**Issue by HaraBeron** [tensorflow/tensorflow#41933]
This is the same issue mentioned [here](https://github.com/tensorflow/tensorflow/issues/41413).

Currently [here](https://colab.research.google.com/github/tensorflow/hub/blob/master/examples/colab/tf2_text_classification.ipynb#scrollTo=zXXx5Oc3pOmN) under 'Loss function and optimizer' it says:

```
model.compile(optimizer='adam',
              loss=tf.losses.BinaryCrossentropy(from_logits=True),
              metrics=['accuracy'])
```

This needs to be corrected to:

`model.compile(optimizer='adam', loss=tf.keras.losses.BinaryCrossentropy(from_logits=True), metrics=[tf.metrics.BinaryAccuracy(threshold=0.0, name='accuracy')])`
